### ISSUE TITLE
API to get the screen resolution

### ISSUE THREAD

**Issue by dnfield** [flutter/flutter#59577]
`ui.window.physicalSize` and `MediaQuery.size` (at least for the root one provided by the framework) provide the _current size_ of the Window object. On mobile platforms, this is typically the same as the screen resolution.

On web and desktop, the window can be trivially resized. If a user wishes to, e.g., size an image based on the maximum resolution of the screen, they don't have a good way to do that right now.

/cc @yjbanov @stuartmorgan for thoughts on severity of this issue.

**Comment by stuartmorgan** [comment]
See https://docs.google.com/document/d/11_4wntz_9IJTQOo_Qhp7QF4RfpIMTfVygtOTxQ4OGHY/edit#heading=h.hn5r6jtglbq6

/cc @gspencergoog

**Comment by dnfield** [comment]
That sounds exactly like it would solve this.

**Comment by gspencergoog** [comment]
Yes, it should.  I separate out the screen from the window, and they have separate update pathways, and you can both query the screen for its resolution/dimensions (and even position relative to other screens), and modify the window's configuration.
